### ISSUE TITLE
Unable to generate Dictionary property with Angular template

### ISSUE THREAD

**Issue by asiffermann** [geeklearningio/gl-swagger-generator#35]
I have the following definition in my Swagger file:
```json
"ClaimsSet": {
    "type": "object",
    "properties": {
        "claims": {
            "type": "object",
            "additionalProperties": {
                "type": "array",
                "items": {
                    "type": "string"
                }
            }
        }
    }
}
```

With the .NET generator, it is properly mapped to a Dictionary<string, string[]>, but with the Angular one, the type is not resolved.
So the generated TypeScript ends with that (and cannot be compiled):
```typescript
/*ClaimsSet*/
export interface ClaimsSet  {

    /**/
    claims: ;

}
```

Is it related to generics' support?

**Comment by sandorfr** [comment]
No just dictionary support missing from the template, something like this should do the trick : https://github.com/geeklearningio/gl-swagger-generator/blob/feature/fix-ts-dictionnary/templates/typescript/angular/1.5/_typeCore.hbs

**Comment by sandorfr** [comment]
fixed in 5a9680cdfd821f3a0d2d785a1c5f1494780b1a40 btw
